tools/audit-docs.js

```diff
@@ -55,18 +79,53 @@ function formatScores(scores) {
   return formattedScores;
 }
 
+
+// Launch the light-server to run tests again
+const bundle = process.argv[2];
+const delay = process.argv[3];
+const bind = 'localhost';
```

Comment:
Rather than binding to localhost, we should bind to `0.0.0.0` (which is pretty safe since this is a test script, not a production server of any kind.